### PR TITLE
[WalletContainer] - add component

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-632

### CODE REVIEW

File: packages/ui/src/1_atoms/Icon/iconSvgPaths.ts [hunk 1]
```diff
@@ -79,4 +79,15 @@ export const IconSvgPaths: Record<IconName, string[]> = {
...
+    'M5.72002 0H3.8294C1.72925 0 0 1.72925 0 3.8294V5.72002H5.72002V0Z',
+    'M0 9.13989H5.72002V14.8599H0V9.13989Z',
+    'M18.2798 23.9519H20.1704C22.2718 23.9519 23.9998 22.2226 23.9998 20.1225C23.9998 20.1189 23.9998 20.114 23.9998 20.1104V18.28H18.2798V23.9519Z',
+    'M9.14014 18.28H14.8602V24H9.14014V18.28Z',
+    'M0 18.28V20.1706C0 22.2708 1.72925 24 3.8294 24H5.72002V18.28H0Z',
+  ],
+  trezor: [
+    'm12.01,0.25c-3.38,0 -6.13,2.63 -6.13,5.86l0,2.19c-1.19,0.21 -2.38,0.48 -2.38,0.84l0,11.46c0,0 0,0.32 0.37,0.47c1.35,0.52 6.67,2.32 7.89,2.73c0.16,0.06 0.21,0.06 0.25,0.06c0.06,0 0.08,0 0.25,-0.06c1.22,-0.41 6.54,-2.21 7.9,-2.73c0.35,-0.14 0.36,-0.45 0.36,-0.45l0,-11.47c0,-0.36 -1.18,-0.65 -2.37,-0.84l0,-2.19c0.02,-3.24 -2.74,-5.86 -6.12,-5.86zm0,2.8c2,0 3.21,1.15 3.21,3.06l0,1.91c-2.24,-0.15 -4.16,-0.15 -6.4,0l0,-1.91c0,-1.92 1.2,-3.06 3.19,-3.06l0,0zm-0.01,7.78c2.79,0 5.13,0.2 5.13,0.57l0,7.16c0,0.1 -0.02,0.12 -0.12,0.16c-0.09,0.04 -4.75,1.64 -4.75,1.64c0,0 -0.19,0.06 -0.25,0.06c-0.06,0 -0.24,-0.07 -0.24,-0.07c0,0 -4.66,-1.61 -4.75,-1.64c-0.1,-0.05 -0.12,-0.06 -0.12,-0.17l0,-7.15c-0.03,-0.37 2.31,-0.56 5.1,-0.56l0,0z',
+  ],
```
Comment (creed-victor) [inline packages/ui/src/1_atoms/Icon/iconSvgPaths.ts]: wallet icon will go from the onboarding lib, so you can remove it from here
```suggestion
```

**Review by creed-victor — CHANGES_REQUESTED** [review]
Remove ledger and trezor icons from here and use any random ones that already exists just as example.

Actual icons for wallets will come from another library and will be lazy loaded, so it's not needed here.

File: packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+export const WalletContainer: FC<WalletContainerProps> = ({
+  name,
+  icon,
+  tooltip,
+  className,
+  dataLayoutId,
+}) => {
+  return (
+    <div
+      data-layout-id={dataLayoutId}
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx]:
```suggestion
      data-action-id={dataLayoutId}
```

File: packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      className={classNames(className, styles.walletContainer)}
+    >
+      <div className="flex items-center justify-center">
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx]: move tailwind classes to css module (build will not work with inline tailwind classes)

```suggestion
      <div className={styles.label}>
```

File: packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        {name}
+        {tooltip && <Icon icon="info" className="ml-1.5" size={10} />}
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx]: no inline tailwind here too

```suggestion
        {tooltip && <Icon icon="info" className={styles.tooltipIcon} size={10} />}
```

**Review by creed-victor — CHANGES_REQUESTED** [review]

File: packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx (hunk 2 above)
Comment (creed-victor) [reply packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx]: my bad, layout-id is correct.

File: packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      className={classNames(className, styles.walletContainer)}
+    >
+      <div className={styles.label}>
+        {name}
+        {tooltip && <Icon icon="info" size={10} />}
+      </div>{' '}
+      {icon && <div className={styles.icon}>{icon}</div>}
+    </div>
+  );
+};
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx]: Change it to button and pass default button props too:

```suggestion
type WalletContainerProps = {
  name: string;
  icon?: ReactNode;
  tooltip?: string;
  dataLayoutId?: string;
} & React.ButtonHTMLAttributes<React.HTMLButtonElement>;

export const WalletContainer: FC<WalletContainerProps> = ({
  name,
  icon,
  tooltip,
  dataLayoutId,
  ...buttonProps
}) => {
  return (
    <button
      data-layout-id={dataLayoutId}
      className={classNames(className, styles.walletContainer)}
      {...buttonProps}
    >
      <div className={styles.label}>
        {name}
        {tooltip && <Icon icon="info" size={10} />}
      </div>{' '}
      {icon && <div className={styles.icon}>{icon}</div>}
    </button>
  );
};
```

**Review by creed-victor — CHANGES_REQUESTED** [review]

File: packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx (hunk 5 above)
Comment (pietro-maximoff) [reply packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx]: ah ok, i thought that we would wrap this container in a button outside where we plan to use it

**Review by creed-victor — APPROVED** [review]

**Pull request merged**